contributors/devel/faster_reviews.md

```diff
@@ -1,218 +1,3 @@
-
-## 9. KISS, YAGNI, MVP, etc
-
-Sometimes we need to remind each other of core tenets of software design - Keep
-It Simple, You Aren't Gonna Need It, Minimum Viable Product, and so on. Adding
-features "because we might need it later" is antithetical to software that
-ships. Add the things you need NOW and (ideally) leave room for things you
-might need later - but don't implement them now.
-
-## 10. Push back
-
-We understand that it is hard to imagine, but sometimes we make mistakes. It's
-OK to push back on changes requested during a review. If you have a good reason
-for doing something a certain way, you are absolutely allowed to debate the
-merits of a requested change. You might be overruled, but you might also
-prevail. We're mostly pretty reasonable people. Mostly.
-
-## 11. I'm still getting stalled - help?!
-
-So, you've done all that and you still aren't getting any PR love? Here's some
-things you can do that might help kick a stalled process along:
-
-   * Make sure that your PR has an assigned reviewer (assignee in GitHub). If
-this is not the case, reply to the PR comment stream asking for one to be
-assigned.
-
-   * Ping the assignee (@username) on the PR comment stream asking for an
-estimate of when they can get to it.
-
-   * Ping the assignee by email (many of us have email addresses that are well
-published or are the same as our GitHub handle @google.com or @redhat.com).
-
-   * Ping the [team](https://github.com/orgs/kubernetes/teams) (via @team-name)
-that works in the area you're submitting code.
-
-If you think you have fixed all the issues in a round of review, and you haven't
-heard back, you should ping the reviewer (assignee) on the comment stream with a
-"please take another look" (PTAL) or similar comment indicating you are done and
-you think it is ready for re-review. In fact, this is probably a good habit for
-all PRs.
-
-One phenomenon of open-source projects (where anyone can comment on any issue)
-is the dog-pile - your PR gets so many comments from so many people it becomes
-hard to follow. In this situation you can ask the primary reviewer (assignee)
-whether they want you to fork a new PR to clear out all the comments. Remember:
-you don't HAVE to fix every issue raised by every person who feels like
-commenting, but you should at least answer reasonable comments with an
-explanation.
-
-## Final: Use common sense
-
-Obviously, none of these points are hard rules. There is no document that can
-take the place of common sense and good taste. Use your best judgment, but put
-a bit of thought into how your work can be made easier to review. If you do
-these things your PRs will flow much more easily.
-
-
-<!-- BEGIN MUNGE: GENERATED_ANALYTICS -->
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/docs/devel/faster_reviews.md?pixel)]()
-<!-- END MUNGE: GENERATED_ANALYTICS -->
+Merged with the [pull requests doc](pull-requests.md).
```

Comment:
Gotcha. I'm okay with this then. Perhaps we leave the analytics bit then, and also target the appropriate part of the doc:
```
pull-requests.md#best-practices-for-faster-reviews
```